shapely/tests/test_predicates.py

```diff
@@ -111,10 +111,20 @@ def test_binary_missing(func):
     assert (~actual).all()
 
 
+def test_binary_empty_result():
+    a = LineString([(0, 0), (3, 0), (3, 3), (0, 3)])
+    b = LineString([(5, 1), (6, 1)])
+    with ignore_invalid(shapely.geos_version < (3, 12, 0)):
+        # Intersection resulting in empty geometries give 'invalid value encountered'
+        # (https://github.com/shapely/shapely/issues/1345)
+        actual = shapely.intersection(a, b)
```

Comment:
```suggestion
        assert shapely.intersection(a, b).is_empty
```

(slight simplification)